finagle-http2/src/main/scala/com/twitter/finagle/http2/transport/AdapterProxyChannelHandler.scala

```diff
@@ -10,6 +10,8 @@ import io.netty.handler.codec.http.HttpClientUpgradeHandler.UpgradeEvent
 import io.netty.handler.codec.http.{HttpObject, LastHttpContent}
 import io.netty.util.ReferenceCountUtil
 import io.netty.util.concurrent.PromiseCombiner
+import io.netty.util.concurrent.ImmediateEventExecutor
+import io.netty.util.concurrent.{Future => NFuture}
```

Comment:
can we call this a NettyFuture instead?  I think that's what we typically do in finagle.